R/utils-shared.R

```diff
@@ -261,7 +261,8 @@ splitDataGaps <- function(data, ts, isDV){
       endGaps <- c(endGaps, endEstimated)
     }
     
-    if(isDV){ ts$time <- flexibleTimeParse(ts$time, timezone = data$reportMetadata$timezone) }
```

Comment:
This line was what was causing the DV data to be rendered at noon instead of on the date lines because flexibleDateTimeParse was adding a time of noon to all of the time fields. Commenting out this line fixed that but it should be revisited after work is done on AQCU-816.